remotable.gemspec

```diff
@@ -15,11 +15,14 @@ Gem::Specification.new do |s|
   s.add_dependency "activerecord"
   s.add_dependency "activesupport"
 
+  s.add_development_dependency "appraisal"
```

Comment:
👏 👏 👏 👏 
I've not heard of this until now. Very cool.

Reply:
I learned about it from the `pluck_map` gem. Seems pretty handy! 😁